test/unit/core.js

```diff
@@ -627,6 +627,7 @@ test("jQuery('html')", function() {
 
 	ok( jQuery("<div></div>")[0], "Create a div with closing tag." );
 	ok( jQuery("<table></table>")[0], "Create a table with closing tag." );
+	ok( jQuery("<tr-d></tr-d>")[0], "Create a dash-delimited tag." );
```

Comment:
Not a problem! I changed as suggested. Please let me know if anything requires adjustments.